examples/pytorch/multiple-choice/train.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+LOG_DIR="./logs"
+OUTPUT_DIR="./outputs"
+mkdir -p $LOG_DIR
+mkdir -p $OUTPUT_DIR
+##    SETTINGS     ## 
+MODEL=$1
+BATCH_SIZE=$2
+log_file="${LOG_DIR}/${model_name}.log"
+output_dir=${3:-"$OUTPUT_DIR/$model_name"}
+gpu_size=$4
```

Comment:
One minor detail: arguments which have default values should be placed last in the argument list. In this case, if you omit `output_dir` but specify `gpu_size`, the script will register your `gpu_size` in the place of `output_dir` (the 3rd arguments), which is unintended. You should either push `gpu_size` as `$3`, or give `gpu_size` a default value.

Reply:
@huonglarne Since it was already merged, in this case, you can push a minor fix commit directly to main branch.

Reply:
@loctxmoreh @yeomni There are more problems to the script other than that. I already fixing it in a new branch and will make a pull request later